index.html

```diff
@@ -28,3 +29,53 @@
         </div>
     </div>
 </div>
+
+{% raw  %}
+<script type="text/javascript">
+    //assume Mustache is loaded
+    //assume Meetup is loaded
+    
+    // Modify from _includes/meetup_cards.html 
+
+    window.onload = function(){
```

Comment:
This should probably use `$(document).ready(function ...)` (or just `$(function ...)`)

Reply:
I cannot use `$(document).ready(function ...)` here since jQuery will be imported after this code. Would you mind if I move `<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>` to `<head>`?